File: CPP Basics/week.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main(){
    int day;
    cout << "Enter number from 0-6" << endl;
    cin >> day;
    switch(day){
        case 0:
           cout <<"Monday";
        break;
        case 1:
          cout <<"Tuesday";
        break;
        case 2:
          cout <<"Wednesday";
        break;
        case 3:
          cout <<"Thursday";
        break;
        case 4:
          cout <<"Friday";
        break;
        case 5:
          cout <<"Saturday";
        break;
        case 6:
          cout <<"Sunday";
        break;
        default:
          cout <<"Invalid Input";
        break;
    }
    return 0;
}

File: Java Basics/week.java
//Wrt\ite a program to print the day of the week based on the Input
// 0- sun. 1-Mon , 2- Tue.....


import java.util.Scanner;
class week{
    public static void main(String[] args){
       System.out.println("Enter Your number");
       Scanner scan=new Scanner(System.in);
       int day=scan.nextInt();
       switch(day){
        case 0:
            System.out.println("Sunday");
            break;
        case 1:
            System.out.println("Monday");
            break;
        case 2:
            System.out.println("Tuesday");
            break;
        case 3:
            System.out.println("Wednesday");
            break;
        case 4:
            System.out.println("Thursday");
            break;
        case 5:
            System.out.println("Friday");
            break;
       case 6:
            System.out.println("Saturday");
            break;
        default:
            System.out.println("Invalid Input");
       }
    }
}
